scripts/zkai_attention_kv_proof_size_profile_gate.py

```
@@ -0,0 +1,880 @@
+    validate_profile_row(rows[1], expected=EXPECTED_TWO_HEAD)
+    validate_scaling_diagnostics(payload.get("scaling_diagnostics"), rows[0], rows[1])
+    if payload.get("profile_gate_commitment") != expected_commitment(payload):
+        raise AttentionKvProofSizeProfileGateError("profile gate commitment drift")
+    if not allow_missing_mutation_summary:
+        validate_mutation_summary(payload)
+
+
+def validate_mutation_summary(payload: dict[str, Any]) -> None:
+    cases = payload.get("mutation_cases")
+    if not isinstance(cases, list) or len(cases) != len(EXPECTED_MUTATION_NAMES):
+        raise AttentionKvProofSizeProfileGateError("mutation case count drift")
```

Ruff (0.15.12)

[warning] Avoid specifying long messages outside the exception class

(TRY003)